apps/storybook/src/Html.stories.tsx

```diff
@@ -46,7 +46,7 @@ export const Container: Story<{
   const [portalContainer, setPortalContainer] = useState<HTMLDivElement>();
 
   return (
-    <>
+    <div style={{ display: 'flex' }}>
```

Comment:
The canvas was progressively growing to fill the height on first render because of the empty `div` below (which was taking up half of the grid on first render and then shrinking).